### ISSUE TITLE
lus covid bug & training information

### ISSUE THREAD

**Issue by martinjaggi** [epfml/disco#355]
in LUS, when training alone, it doesn't start but showing me

```
Error: model's task does exist locally
    at Proxy.taskTitle (Validation.vue?494b:100:15)
    at Validation.vue?4593:68:60
```
<img width="390" alt="Screen Shot 2022-07-05 at 10 29 33" src="https://user-images.githubusercontent.com/4621895/177285617-6f337c1b-7bba-4875-bbc3-ed5f33b72d96.png">

- [x] if it fails that error should actually also be shown to the user in the UI, not only in the dev console

- [x] another issue with the UI: showing avg/current number of participants doesn't make sense for training alone. can remove that (and keep it only for decentralized and federated)

- [x] in decentralized UI, show avg/current number of participants. # people helped is unclear (got help from or to them?). # averaging should also be removed, can be just `number of rounds` instead (same as in federated)

- [x] either fix or move LUS task

**Comment by s314cy** [comment]
i was not able to reproduce the console error you got, did you try to test the model at a certain point?

**Comment by martinjaggi** [comment]
it might have been that i first clicked on test, where i did nothing, then went back to train. let me try again now

**Comment by martinjaggi** [comment]
this is chrome on a mac. i just go on task on the left, connect a handful of the example pngs for both classes, then click train alone.

now the chrome console says in `topology.js:508`  
```
Uncaught (in promise) Error: Input 0 is incompatible with layer dense_Dense1: expected axis -1 of input shape to have value 1000 but got shape 2,224,224,3.
    at WT.assertInputCompatibility (topology.js:508:31)
    at topology.js:684:18
    at lS (common.js:39:21)
    at WT.apply (topology.js:654:16)
    at WI (executor.js:283:47)
    at a (training.js:1073:33)
    at engine.js:328:22
    at Ds.scopedRun (engine.js:338:25)
    at Ds.tidy (engine.js:327:21)
    at engine.js:858:91
```    
(and round remains always at zero)

**Comment by s314cy** [comment]
> this is chrome on a mac. i just go on task on the left, connect a handful of the example pngs for both classes, then click train alone.
> 
> now the chrome console says in `topology.js:508`
> 
> ```
> Uncaught (in promise) Error: Input 0 is incompatible with layer dense_Dense1: expected axis -1 of input shape to have value 1000 but got shape 2,224,224,3.
>     at WT.assertInputCompatibility (topology.js:508:31)
>     at topology.js:684:18
>     at lS (common.js:39:21)
>     at WT.apply (topology.js:654:16)
>     at WI (executor.js:283:47)
>     at a (training.js:1073:33)
>     at engine.js:328:22
>     at Ds.scopedRun (engine.js:338:25)
>     at Ds.tidy (engine.js:327:21)
>     at engine.js:858:91
> ```
> 
> (and round remains always at zero)

oh right, this task was implemented by @MarcelTorne a long time ago

since we don't apply input resizing anymore, we get a shape mismatch with regular images (3 channels vs. flat array expected), we should either display a clear error message to the user or consider the task as stale for now

**Comment by martinjaggi** [comment]
ok. maybe not too hard to add the resizing again to this task? 

it's kind of a nice task as images are available and easy to run for a test (2 classes, less drag and drop work than mnist).

or delete, or update the description / move to bottom of task list (now on top)

**Comment by s314cy** [comment]
turns out the server does not provide the correct statistics
